README.md

```diff
@@ -9,30 +9,40 @@
 
 > Note: Looking for collaborators. [Want to help improve ts-jest?](https://github.com/kulshekhar/ts-jest/issues/223)
 
+`ts-jest` is a TypeScript preprocessor for Jest with source map support and 
+sensible defaults that quickly allows you to start testing TypeScript with Jest.
 
```

Comment:
I think sensible defaults work well, but I have no strong feelings, so I'll just change it 👍